recipe/meta.yaml

```diff
@@ -43,17 +43,11 @@ requirements:
     - qhull
     - qhull-static  # [not win]
     - console_bridge
+    - libsdformat
```

Comment:
fyi `libsdformat` installs sdformat 9, while following releases of `sdformat` are released as `libsdformat10`, `libsdformat11`, etc etc. till `libsdformat14` (each package is side-by-side installable in the same environment). Probably we for each `libsdformatX` package we could also add a `libsdformat==X` package that depends on it to support simply depending on `libsdformat`, but this is not done at the moment.

Reply:
Thank for the information.

This is not creating any issue with the sdf headers (sdf/sdf.hpp) ?

Anyway, we always have tested pinocchio against libsdformat, do you recommend we use libsdformat14 ?

Reply:
> This is not creating any issue with the sdf headers (sdf/sdf.hpp) ?

No, Gazebo libraries are specifically designed to be side-by-side installable (even on apt), as the headers are installed in different directories:
* `include/sdformat-9.8/` for sdformat 9: https://conda-metadata-app.streamlit.app/?q=conda-forge%2Flinux-64%2Flibsdformat-9.8.0-h70f8a9e_6.conda
* `include/gz/sdformat14/` for sdformat 14: https://conda-metadata-app.streamlit.app/?q=conda-forge%2Flinux-64%2Flibsdformat14-14.5.0-hb243517_0.conda

The only problem occurs if you try to mix two different SDF libraries in the same target/compilation unit. 

> Anyway, we always have tested pinocchio against libsdformat, do you recommend we use libsdformat14 ?

If you tested sdformat 9 and it is working fine for your use case, I think you can use it. The only thing that comes to my mind is that on apt/ros there is no sdformat9 available, so if you package there pinocchio you are going to use sdformat that is available there.

Reply:
We will use sdformat14 since it seem to be the default one in ros humble.